proposals/description/fip_50.ts

```diff
@@ -0,0 +1,101 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_50: ProposalDescription = {
+  title: 'FIP-50: Yield improvements',
+  commands: [
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantPCVController(address)',
+      arguments: ['{ratioPCVControllerV2}'],
+      description: 'Grant PCV Controller to RatioPCVControllerV2'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantRole(bytes32,address)',
+      arguments: ['0x471cfe1a44bf1b786db7d7104d51e6728ed7b90a35394ad7cc424adf8ed16816', '{optimisticTimelock}'],
+      description: 'Grant SWAP_ADMIN_ROLE to the OA'
+    },
+    {
+      target: 'feiLusdLBPSwapper',
+      values: '0',
+      method: 'exitPool(address)',
```

Comment:
Do we burn the ~1M FEI that will be also sent on the Timelock along with the 99M LUSD ? They will count as user-circulating otherwise. Not sure we can know the exact amount that will land there though (someone could swap in the LUSD LBP)

Reply:
Sending to OA Timelock